emsdk.py

```diff
@@ -191,15 +191,14 @@ def to_unix_path(p):
   return p.replace('\\', '/')
 
 
-def emsdk_path():
-  return to_unix_path(os.path.dirname(os.path.realpath(__file__)))
+emsdk_path = to_unix_path(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
would it make sense for this to be all caps too?

Reply:
Maybe yes... I did do that in the original version of this change but I can't remember why I changed back.

Reply:
Done.. this file is not consistent this respect ... :(